sdk/src/main/java/com/horizen/account/api/rpc/types/ForwardTransfersView.java

```diff
@@ -0,0 +1,61 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import com.horizen.account.proposition.AddressProposition;
+import com.horizen.account.utils.MainchainTxCrosschainOutputAddressUtil;
+import com.horizen.account.utils.ZenWeiConverter;
+import com.horizen.block.MainchainTxForwardTransferCrosschainOutput;
+import com.horizen.serialization.Views;
+import com.horizen.transaction.mainchain.ForwardTransfer;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.TreeMap;
+
+@JsonView(Views.Default.class)
+public class ForwardTransfersView {
+    private final TreeMap<Integer, ForwardTransferData> forwardTransfers = new TreeMap();
+
+    public ForwardTransfersView(List<ForwardTransfer> transactions, boolean noPrefix) {
+        for (int i = 0; i < transactions.size(); i++) {
+            MainchainTxForwardTransferCrosschainOutput ftOutput = transactions.get(i).getFtOutput();
+            var to = "";
+            var value = "";
+            AddressProposition address = new AddressProposition(
+                    MainchainTxCrosschainOutputAddressUtil.getAccountAddress(ftOutput.propositionBytes()));
+            BigInteger weiValue = ZenWeiConverter.convertZenniesToWei(ftOutput.amount());
+            if (noPrefix) {
+                to = Numeric.toHexStringNoPrefix(address.address());
+                value = String.valueOf(weiValue);
+            } else {
+                to = Numeric.toHexString(address.address());
+                value = Numeric.toHexStringWithPrefix(weiValue);
+            }
+            forwardTransfers.put(i, new ForwardTransferData(to, value));
+        }
+    }
+
+    public TreeMap<Integer, ForwardTransferData> getForwardTransfers() {
```

Comment:
why do we need to use Map here?
We pass `List<ForwardTransfer> transactions` -> so the result can be List<ForwardTransferData> to be sure that the result has a proper order of FTs.